### PR TITLE
V1.0 fixes

### PR DESCRIPTION
**Pull request by squeed**
Signed-off-by: Casey Callendrello <cdc@redhat.com>

### CODE REVIEW

File: content/plugins/v1.0/_index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: Plugins Overview
+description: "plugins/README.md"
+date: 2020-11-02
+toc: true
+draft: false
+weight: 200
+cascade:
+  version: current
```
Comment (nate-double-u) [inline content/plugins/v1.0/_index.md:9]: Need to change the version here or the menu system won't work correctly.
```suggestion
  version: v1.0
```

**Review by nate-double-u — CHANGES_REQUESTED** [review]
Just one note inline, otherwise /lgtm

File: content/plugins/v1.0/_index.md (hunk 1 above)
Comment (squeed) [reply content/plugins/v1.0/_index.md:9]: oh, good catch1

**Review by nate-double-u — APPROVED** [review]
Looks good to me!

**Pull request merged**